lib/mvi/responses/profile_parser.rb

```diff
@@ -91,6 +97,21 @@ def build_mvi_profile(patient)
         )
       end
 
+      def log_inactive_mhv_ids(mhv_ids, active_mhv_ids)
+        if (mhv_ids - active_mhv_ids).present?
+          log_message_to_sentry('Inactive MHV correlation IDs present', :info,
+                                extra_context: { ids: mhv_ids })
+        end
+        unless active_mhv_ids.include?(mhv_ids.first)
+          log_message_to_sentry('Returning inactive MHV correlation ID as first identifier', :warn,
+                                extra_context: { ids: mhv_ids })
+        end
+        if active_mhv_ids.size > 1
+          log_message_to_sentry('Multiple active MHV correlation IDs present', :info,
+                                extra_context: { ids: active_mhv_ids })
+        end
+      end
```

Comment:
LOL this is going to be a lot of sentry warnings potentially, right? In any case, lets give this a shot. I like it.